.github/workflows/kubernetes-controller.yml

```
@@ -161,4 +163,13 @@ jobs:
         shell: bash
         run: |
-          docker system prune --force
\ No newline at end of file
+          docker system prune --force
+
+  conformance:
+    name: Conformance
+    needs: publish_latest
+    uses: ./.github/workflows/conformance.yml
+    with:
+      toolkit_chart: "oci://ghcr.io/open-component-model/kubernetes/controller/chart/ocm-k8s-toolkit"
```

actionlint (1.7.11)

[error] input "toolkit_chart" is not defined in "./.github/workflows/conformance.yml" reusable workflow. defined inputs are "cli_image", "toolkit_image"

(workflow-call)